src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -2,31 +2,36 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
-    public int total = -1;
+    private static final int DEFAULT_RETURN_VALUE = -1;
+    private static final int FULL_STACK = 11;

```

Comment:
This constant is related to MAX_STACK_SIZE . Changing only one leads to an error (inconsistency).